src/cmipcite/citations.py

```diff
@@ -474,8 +516,14 @@ def get(  # noqa: PLR0913
     (for CMIP, this collection of files
     is for a single variable sampled at a single frequency and spatial sampling
     from a single model running a single experiment).
-    All datasets from a single model or from a single experiment (and model)
-    are grouped under a DOI, associated with the dataset's PID.
+    For a given PID, we can retrieve the associated DOI.
+    However, there are multiple possibilities for the retrieved DOI.
+    These vary based on the 'grouping level' of the DOI.
+    At the moment, as far as we know, there are two grouping levels:
+    grouping based on the model that provided the data
+    or grouping based on the model that provided the data
+    and the experiment the dataset came from.
+    The `doi_level` controls which DOI grouping level you get.
```

Comment:
Needs to be checked depending on where we land on the meaning and naming of these groups